src/Smoothie.js

```diff
@@ -1,15 +1,37 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
+import './Smoothie.css';
 
 export default class Smoothie extends Component {
   constructor(props) {
     super(props)
+    this.state = {
+      isSelected: false
+    }
   }
 
+  expandRecipe = () => {
+    const {isSelected} = this.state
+    this.setState({isSelected: !isSelected})
+    this.props.showRecipe(this.props)
+    console.log(this.props.recipe)
+  }
+  
   render = () => {
+    const {isSelected} = this.state
     return (
       <div>
-        {this.props.name}
-        <img src={this.props.img}/>
+        {!isSelected && 
+          <div className='smoothie' onClick={this.expandRecipe}>
+            <h3>{this.props.name}</h3>
```

Comment:
Destructure this.props to stay consistent

Reply:
Let's wait and do this locally, that way we can go through all files and make sure consistency is there.